addons/adminMenu/functions/fnc_uihook_enableMissionTesting.sqf

```diff
@@ -0,0 +1,9 @@
+#include "script_component.hpp"
+
+TRACE_1("params",_this);
+
+[QGVAR(missionTestingAddAction),[""]] call CBA_fnc_globalEvent;
+
+EGVAR(missionTesting,missionTestingActive) = true;
```

Comment:
Does this change globally? What happens if two people enable mission testing? Will it double add actions, and will that break anything?